### PR TITLE
Tcd 1539

### PR DESCRIPTION
**Pull request by spider-sauce**
### Description

Documentation refactor based on [slack thread](https://saucelabs.slack.com/archives/G01L52RRAVD/p1621315598005000)
JIRA Issue:[TCD-1539]( https://saucedev.atlassian.net/browse/TCD-1539)
### Motivation and Context

Previous doc only detailed how to run a script against Sauce Labs rather than cover a more detailed approach with:
* How to set up / plug in the `axe-core` library and work with the accessibility feature
* How to enable the Integration in Sauce Labs UI - TODO
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by lroslonek — CHANGES_REQUESTED** [review]

File: docs/basics/integrations/deque.md [hunk 1]
```diff
@@ -9,128 +9,258 @@ keywords:
...
+    connectionRetryTimeout: 120000,
+    connectionRetryCount: 3,
+    services: ['chromedriver'],
+    framework: 'mocha',
+    reporters: ['spec'],
+    mochaOpts: {
+        ui: 'bdd',
+        timeout: 60000
+    },
+}
```
[the remark this answers is not in the document]
Comment (spider-sauce) [reply docs/basics/integrations/deque.md:76]: That's fair. So If I'm to remove the config object, should I remove all mention of `wdio` and only use psuedocode/code snippets to keep it framework agnostic?

File: docs/basics/integrations/deque.md [hunk 2]
```diff
@@ -9,128 +9,258 @@ keywords:
...
+
+## Set Up Dependencies
+
+Add the following dependencies to your `package.json` file:
+
+* `axe-core` for Accessibility tests (`a11y` results)
+* `sauce-service` for the WebdriverIO plugin service
+
+```js
+"@axe-core/webdriverio": "4.1.2-alpha.106"
```
[the remark this answers is not in the document]
Comment (spider-sauce) [reply docs/basics/integrations/deque.md:43]: same thing here ☝️ , should I only reference the general `axe-core` [npm package](https://www.npmjs.com/package/axe-core) and maybe provide links to the specific bindings?

File: docs/basics/integrations/deque.md [hunk 3]
```diff
@@ -9,128 +9,258 @@ keywords:
...
+config.user = process.env.SAUCE_USERNAME;
+config.key = process.env.SAUCE_ACCESS_KEY;
+config.region = process.env.REGION || 'us';
+```
+
+Visit the [WebDriverIO documentation](https://webdriver.io/docs/sauce-service.html) for further details.
+:::
+
+### Set Browser Options
+In order to test on multiple different browsers and platforms, add the configuration options in the `capabilities` object.
```
[the remark this answers is not in the document]
Comment (spider-sauce) [reply docs/basics/integrations/deque.md:169]: If we remove the mention of `wdio` would it still make sense to cover `chromedriver` and the like? If not, should we remove this section too?

File: docs/basics/integrations/deque.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (spider-sauce) [reply docs/basics/integrations/deque.md]: Removed and now I link to the wdio docs.

File: docs/basics/integrations/deque.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (spider-sauce) [reply docs/basics/integrations/deque.md]: Fixed

File: docs/basics/integrations/deque.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (spider-sauce) [reply docs/basics/integrations/deque.md]: Added a section for using local drivers, is there anything else needed for axe to work with the driver objects?

File: docs/basics/integrations/deque.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lroslonek) [reply docs/basics/integrations/deque.md]: it's not really framework agnostic, but it's framework's configuration agnostic. Other places where you mention `wdio` are ok, but this one is a specific configuration, which is not required to run accessibility test

File: docs/basics/integrations/deque.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lroslonek) [reply docs/basics/integrations/deque.md]: I think I was wrong with this one. I noticed that Deque is also referring to the alpha channel in their docs (https://www.deque.com/saucelabs/get-started/), thus I think we should also. Sorry!

File: docs/basics/integrations/deque.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lroslonek) [reply docs/basics/integrations/deque.md]: no, I guess it's ok right now

**Comment by spider-sauce** [comment]
@lroslonek I fixed the suggested changes. If it's ok with you I'm going to merge this so we can replace the current customer-facing doc with this one. If there's further changes/need for further iteration we can create a new ticket.

**Pull request merged**